### PR TITLE
NAS-109606 / 21.04 / Fix creating replication LOCAL tasks

### PR DESCRIPTION
**Pull request by denysbutenko**

### CODE REVIEW

File: src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts [hunk 1]
```diff
@@ -1190,11 +1189,13 @@ export class ReplicationFormComponent {
         if (data['direction'] == 'PUSH') {
             for (let i = 0; i < data['source_datasets_PUSH'].length; i++) {
                 if (_.startsWith(data['source_datasets_PUSH'][i], '/mnt/')) {
+                    console.log("data['source_datasets_PUSH'][i]", data['source_datasets_PUSH'][i], data['source_datasets_PUSH'][i].substring(5))
```
Comment (AlexKarpov98) [inline src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts:1192]: remove debug code

Comment (denysbutenko) [reply src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts]: removed, thanks 👍🏻

File: src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts [hunk 2]
```diff
@@ -1190,11 +1189,13 @@ export class ReplicationFormComponent {
...
                     data['source_datasets_PUSH'][i] = data['source_datasets_PUSH'][i].substring(5);
                 }
             }
-            data['source_datasets'] = _.filter(Array.isArray(data['source_datasets_PUSH']) ? _.cloneDeep(data['source_datasets_PUSH']) : _.cloneDeep(data['source_datasets_PUSH']).split(',').map(_.trim));
-            data['target_dataset'] = typeof data['target_dataset_PUSH'] === 'string' ? _.cloneDeep(data['target_dataset_PUSH']) : _.cloneDeep(data['target_dataset_PUSH']).toString();
+            data['source_datasets'] = _.filter(Array.isArray(data['source_datasets_PUSH']) ? _.cloneDeep(data['source_datasets_PUSH']) : _.cloneDeep(data['source_datasets_PUSH']));
```
Comment (AlexKarpov98) [inline src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts:1195]: seems to be a mistake.

Comment (denysbutenko) [reply src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts]: updated, does it better now?

**Review by AlexKarpov98 — APPROVED** [review]
LGTM

**Pull request merged**